citer.py

```diff
@@ -156,14 +159,16 @@ def get_settings(setting, default):
             if setting == 'bibtex_file':
                 window = sublime.active_window()
                 ref_dir = os.path.dirname(window.project_file_name())
-                result = ref_dir + '/' + project_data['bibtex_file']
+                filename = project_data['bibtex_file']
+                result = ref_dir + '/' + filename.replace('\'', '')
```

Comment:
I think this will fail on windows - need to use `os.path.join` (or work with pathlib but that's probably overkill)